src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -202,11 +209,17 @@ def _alloc_node(self, node, requested_cores, requested_gpus,
 
         # we can serve the partial or full request - alloc the chunks we need
         # FIXME: chunk gpus, too?
+
+        # MING: We don't really need to include chunk in the arithmetic if
+        #       they cancel out right?
         alloc_cores = min(requested_cores, free_cores) / chunk * chunk
```

Comment:
Hmm, a comment is needed it seems :)  Assume the following: `requested_cores=10, free_cores=8, chunk=5` - what is returned if that term (`/chunk*chunk`) is left off?  Why?  Is there a better way to achieve this?
Please come up with a comment to clarify this.  No need to do that before the call, I'll bring this up today.  Nice catch!

Reply:
Oh right, min(10, 8)/5 = 1. I forgot that the real numbers is not a field in computer-land ;)